PWGHF/TableProducer/candidateSelectorLbToLcPi.cxx

```diff
@@ -142,6 +143,15 @@ struct HfCandidateSelectorLbToLcPi {
       return false;
     }
 
+    // distance between Lb and Lc decay
+    float diffXVert = hfCandLb.xSecondaryVertex() - hfCandLc.xSecondaryVertex();
+    float diffYVert = hfCandLb.ySecondaryVertex() - hfCandLc.ySecondaryVertex();
+    float diffZVert = hfCandLb.zSecondaryVertex() - hfCandLc.zSecondaryVertex();
+    float vertexDistance = sqrt(diffXVert * diffXVert + diffYVert * diffYVert + diffZVert * diffZVert);
```

Comment:
You can avoid typing the full calculation by using `RecoDecay::sqrtSumOfSquares`.
If you prefer the explicit calculation, please use `std::sqrt`.